ggml/src/ggml-backend-reg.cpp

```diff
@@ -155,6 +159,11 @@ static void * dl_get_sym(dl_handle * handle, const char * name) {
     return dlsym(handle, name);
 }
 
+static const char *dl_error() {
```

Comment:
```suggestion
static const char * dl_error() {
```

Reply:
Thanks for remark, fixed and rebased to latest version. May require another approval to run CI.